scripts/res/Semver.res

```diff
@@ -0,0 +1,75 @@
+type preRelease = Alpha(int) | Beta(int) | Dev(int) | Rc(int)
```

Comment:
Is there a reason not to use https://www.npmjs.com/package/semver?

Reply:
I just copy-pasted the code from the website, no idea why we did this in the first place.